src/ui/src/js/controllers/admin_garden.js

```diff
@@ -42,6 +44,19 @@ export default function adminGardenController(
     GardenService.syncGardens()
   }
 
+  $scope.createGarden = function() {
+      if ($scope.garden_name != "" & $scope.garden_name != null){
+        GardenService.createGarden({"name":$scope.garden_name, "status":"NOT_CONFIGURED"});
+        $state.go('base.garden_view',
+              {
+                'name': $scope.garden_name,
+              }
+            );
+      }
+      $scope.createGardenFormHide = true;
```

Comment:
Does this actually do anything? If this condition is hit, then the field was not populated, so isn't that an error condition? You should do an else statement and throw an validation alert that they didn't provide an Garden Name for the create function. Also, don't hide the form during the failure case.